src/main.cpp

```diff
@@ -5,27 +5,29 @@
 
 #include <SFML/Graphics.hpp>
 #include <cmath>
+#include <chrono>
 #include <iostream>
 #include "Grid.hpp"
 
-
 namespace {
 
+const sf::Vector2<uint32_t> screen_dimensions { 1420, 1000 };
 const int SCREEN_WIDTH = 1420;
 const int SCREEN_HEIGHT = 1000;
 
 void run(char** argv) {
     
-    sf::Clock clock;
-    sf::Time elapsed;
+    //init clock
+    std::chrono::time_point<std::chrono::system_clock> start{}, end{};
```

Comment:
Don't need two time points for a game loop, one is sufficient to compute delta time between frames.

Reply:
Why would you want to limit FPS? Instead turn on vsync.